### DISCUSSION TITLE
你猜我为什要花10元巨资买下PCL2的快照使用权

### DISCUSSION THREAD

**Discussion by huhujieshuo** [Meloong-Git/PCL#2267] [讨论]
本来想要加个PCL2的官方启动器的，没想到停服了（早期知道可以解锁隐藏主题，还买了个XGP)

**Comment by iPodtouch6557** [comment]
我是斥23.33人民币的巨资买的，差点还因为防火墙问题不能更新……
买了快照可以让自己在这个社区的参与感更强，我是这么买的，因为我以前一直被windefender困扰导致无法更新
